lib/forms/responses/response.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'common/client/concerns/service_status'
+require 'common/models/base'
+
+module Forms
+  module Responses
+    ##
+    # Model for Forms responses. Body is passed straight through from the service
+    # with a validation check that it matches the expected schema.
+    #
+    # @!attribute body
+    #   @return [Hash] Validated response body.
+    # @!attribute status
+    #   @return [Integer] The HTTP status code.
+    #
+    class Response < Common::Base
+      attribute :body, String
+      attribute :status, Integer
+
+      def initialize(status, body, schema_name)
+        self.body = body if json_format_is_valid?(body, schema_name)
+        self.status = status
+      end
+
+      private
+
+      def json_format_is_valid?(body, schema_name)
+        schema_path = Rails.root.join('lib', 'forms', 'schemas', "#{schema_name}.json").to_s
```

Comment:
Would this ever be anything but `forms.json` for you?

Reply:
Not right now, but potentially. That health-check route may come into play later but I think this PR is large enough already.